### PR TITLE
Nodejs/xiaolei/td 2979

### PR DESCRIPTION
**Pull request by xleili**
nodejs test case of 1970

### CODE REVIEW

File: src/connector/nodejs/test/test1970.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  c1.execute("insert into "+dbname+"."+tbname+" values(1628928479484,15)");
+
+// Select
+console.log("select * from "+dbname+"."+tbname);
+c1.execute("select * from "+dbname+"."+tbname);
+
+var d = c1.fetchall();
+console.log(c1.fields);
+for (let i=0 ; i<d.length;i++)
+console.log(d[i][0].valueOf());
```
Comment (sangshuduo) [inline src/connector/nodejs/test/test1970.js]: no indent?

File: src/connector/nodejs/test/test1970.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  c1.execute("insert into "+dbname+"."+tbname+" values('2021-08-14 08:07:59.484',5)");
+
+// Select
+console.log("select * from "+dbname+"."+tbname);
+c1.execute("select * from "+dbname+"."+tbname);
+
+var d = c1.fetchall();
+console.log(c1.fields);
+for (let i=0 ; i<d.length;i++)
+console.log(d[i][0].valueOf());
```
Comment (sangshuduo) [inline src/connector/nodejs/test/test1970.js]: no indent?

File: src/connector/nodejs/test/test1970.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+//0  1970-01-01 08:00:00.00
+//1628928479484  2021-08-14 08:07:59.484
+console.log("insert into "+dbname+"."+tbname+" values(-28800001,11)");
+c1.execute("insert into "+dbname+"."+tbname+" values(-28800001,11)");
+
+console.log("insert into "+dbname+"."+tbname+" values(-28800000,12)");
+c1.execute("insert into "+dbname+"."+tbname+" values(-28800000,12)");
+
+console.log("insert into "+dbname+"."+tbname+" values(-1,13)");
+  c1.execute("insert into "+dbname+"."+tbname+" values(-1,13)");
```
Comment (sangshuduo) [inline src/connector/nodejs/test/test1970.js]: why indent?

File: src/connector/nodejs/test/test1970.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+//1970-01-01 08:00:00.000a
+//1628928479484  2021-08-14 08:07:59.484
+console.log("insert into "+dbname+"."+tbname+" values('1969-12-31 23:59:59.999',1)");
+c1.execute("insert into "+dbname+"."+tbname+" values('1969-12-31 23:59:59.999',1)");
+
+console.log("insert into "+dbname+"."+tbname+" values('1970-01-01 00:00:00.000',2)");
+c1.execute("insert into "+dbname+"."+tbname+" values('1970-01-01 00:00:00.000',2)");
+ 
+console.log("insert into "+dbname+"."+tbname+" values('1970-01-01 07:59:59.999',3)");
+  c1.execute("insert into "+dbname+"."+tbname+" values('1970-01-01 07:59:59.999',3)");
```
Comment (sangshuduo) [inline src/connector/nodejs/test/test1970.js]: why indent?

**Review by xleili — COMMENTED** [review]
ident fixed

File: src/connector/nodejs/test/test1970.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+//c1.execute('use ' + dbname)
+//1969-12-31 23:59:59.999
+//1970-01-01 00:00:00.000
+//1970-01-01 07:59:59.999
+//1970-01-01 08:00:00.000a
+//1628928479484  2021-08-14 08:07:59.484
+
+console.log("insert into "+dbname+"."+tbname+" values('1969-12-31 23:59:59.999',1)");
+c1.execute("insert into "+dbname+"."+tbname+" values('1969-12-31 23:59:59.999',1)");
+
```
Comment (zitsen) [inline src/connector/nodejs/test/test1970.js]: better to use like this:

```js
let sql = '';
console.log(sql);
c1.execute(sql);
```

File: src/connector/nodejs/test/test1970.js [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+c1.execute("insert into "+dbname+"."+tbname+" values('2021-08-14 08:07:59.484',5)");
+
+// Select
+console.log("select * from "+dbname+"."+tbname);
+c1.execute("select * from "+dbname+"."+tbname);
+
+var d = c1.fetchall();
+console.log(c1.fields);
+for (let i=0 ; i<d.length;i++)
+    console.log(d[i][0].valueOf());
```
Comment (zitsen) [inline src/connector/nodejs/test/test1970.js]: indent by 2 space across the nodejs project.

**Review by zitsen — APPROVED** [review]

File: src/connector/nodejs/test/test1970.js (hunk 5 above)
Comment (sangshuduo) [reply src/connector/nodejs/test/test1970.js]: yes, that's DRY (Don't Repeast Yourself) principle. @xleili

Comment (xleili) [reply src/connector/nodejs/test/test1970.js]: thank you. I got it.

**Review by sangshuduo — CHANGES_REQUESTED** [review]
need to apply DRY principle

**Review by xleili — COMMENTED** [review]
applied dry principle

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by xleili — COMMENTED** [review]
add ci

**Pull request merged**